freehep maven dependency issues

Started probably yesterday ...

[ERROR] Failed to execute goal on project clas12rec: Could not collect dependencies for project org.jlab.clas:clas12rec:pom:10.1.2-SNAPSHOT
[ERROR] Failed to read artifact descriptor for org.freehep:freehep-jminuit:jar:1.0.2
[ERROR] 	Caused by: The following artifacts could not be resolved: org.freehep:freehep-jminuit:pom:1.0.2 (absent): Could not transfer artifact org.freehep:freehep-jminuit:pom:1.0.2 from/to freehep-repo-public (https://srs.slac.stanford.edu/nexus/content/groups/freehep-maven2-public/): Connection reset

--- a/common-tools/coat-lib/pom.xml
+++ b/common-tools/coat-lib/pom.xml
@@ -14,11 +14,12 @@
     <repository>
       <id>jnp-build</id>
       <url>https://clasweb.jlab.org/jhep/maven</url>
-    </repository>
+  </repository>
     <repository>
      <id>freehep-repo-public</id>
-     <url>https://srs.slac.stanford.edu/nexus/content/groups/freehep-maven2-public/</url>
-    </repository>
+     <url>https://clasweb.jlab.org/clas12maven/</url>
+     <!--<url>https://srs.slac.stanford.edu/nexus/content/groups/freehep-maven2-public/</url>-->
+  </repository>
   </repositories>
 
   <dependencies>

--- a/maven-settings.xml
+++ b/maven-settings.xml
@@ -9,10 +9,17 @@ that is pulling it in.
     <mirrors>
         <mirror>
             <id>new-freehep</id>
-            <mirrorOf>freehep</mirrorOf>
-            <name>freehep</name>
+            <mirrorOf>freehep-jminuit</mirrorOf>
+            <name>freehep-jminuit</name>
+            <url>https://clasweb.jlab.org/clas12maven</url>
             <!--<url>https://java.freehep.org/maven2</url>-->
-            <url>https://srs.slac.stanford.edu/nexus/content/groups/freehep-maven2-public/</url>
+            <!--<url>https://srs.slac.stanford.edu/nexus/content/groups/freehep-maven2-public/</url>-->
+        </mirror>
+        <mirror>
+            <id>new-freehep-repo-public</id>
+            <mirrorOf>freehep-repo-public</mirrorOf>
+            <name>freehep-repo-public</name>
+            <url>https://clasweb.jlab.org/clas12maven</url>
         </mirror>
     </mirrors>
 </settings>

